pkg/apis/playlist/v0alpha1/conversions.go

```diff
@@ -0,0 +1,47 @@
+package v0alpha1
+
+import (
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/grafana/grafana/pkg/services/playlist"
+)
+
+type namespaceMapper = func(orgId int64) string
+
+func orgNamespaceMapper(orgId int64) string {
+	if orgId == 1 {
+		return "default"
+	}
+	return fmt.Sprintf("org-%d", orgId)
+}
+
+func convertToK8sResource(v *playlist.PlaylistDTO, namespacer namespaceMapper) *Playlist {
```

Comment:
do we need something to go the opposite way?

Reply:
not yet, but we will when we support write!